plugins/ipam/dhcp/README.md

```diff
@@ -1,4 +1,4 @@
 
 This document has moved to the [containernetworking/cni.dev](https://github.com/containernetworking/cni.dev) repo.
 
-You can find it online here: https://cni.dev/plugins/ipam/dhcp/
+You can find it online here: https://cni.dev/plugins/current/ipam/dhcp/
```

Comment:
What about released versions?

Reply:
Do you mean the docs of the older versions (0.6 - 0.9)? I supposed that the "current" version of the docs was a reasonable default.

Reply:
How about using a template `${VERSION:-current}`?

Reply:
Not sure if I follow correctly. The change is in the markdown text linking to the docs URL, and I don't know how including variables in text/links is done in markdown.

I don't think I explained it properly in the issue: pages such as [this k8s doc](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) link to the github sub-page of a plugin ex. [portmap](https://github.com/containernetworking/plugins/tree/master/plugins/meta/portmap). That page links to the online docs and that link is currently 404 due to the versioning issue, so I was trying to link to the correct page. 

The docs for different versions are handled by the "Versions" dropdown at the [CNI page](https://www.cni.dev/plugins/current/).

Thanks!